lib/galaxy/authnz/psa_authnz.py

```diff
@@ -119,6 +119,10 @@ def __init__(self, provider, oidc_config, oidc_backend_config, app_config: Galax
         self.config[setting_name("USER_MODEL")] = "models.User"
         # Use a custom auth pipeline if configured.
         auth_pipeline = app_config.get("oidc_auth_pipeline", AUTH_PIPELINE)
+        # Add extra steps to the auth pipeline if configured.
+        pipeline_extra = app_config.get("oidc_auth_pipeline_extra", [])
```

Comment:
No need to `get` (same applies to the line above): since this is in the schema, the attribute on the config object is created automatically at startup), you can just use `app_config.oidc_auth_pipeline_extra`.